Script/Packages/Atomic/Scene.json

```diff
@@ -8,7 +8,8 @@
 	"excludes" : {
 		"Scene" : {
 			"GetComponent" : ["unsigned"],
-			"MarkReplicationDirty" : ["Node"]
+			"MarkReplicationDirty" : ["Node"],
+			"GetRequiredPackageFiles" : []
 		}
```

Comment:
GetRequiredPackageFiles() was getting an error trying to convert a PackageFile pointer to an AObject something, which won't happen. So I excluded it, the loss from not having it scripted will be minimal. It is only used in the client-server scene+packagefile replication internals, which are also not exposed to scripting.